client/lib/plans/features-list.js

```diff
@@ -1078,7 +1078,10 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_BACKUP_REALTIME_V2 ]: {
 		getSlug: () => constants.FEATURE_BACKUP_REALTIME_V2,
-		getTitle: () => i18n.translate( 'Automated real-time site backups' ),
+		getTitle: () =>
+			( {
+				i5: i18n.translate( 'Automated real-time backups' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.